101-bundle-json.md

```diff
@@ -196,6 +197,20 @@ The following OPTIONAL fields MAY be attached to an invocation image:
   - `os`: The operating system of the image
 - `mediaType`: The media type of the image
 
+### Resolving Image References
+
+With invocation images and images that use the OCI image format, the referenced image is of the format `REGISTRY/IMAGE:TAG@DIGEST`.
+
+`REGISTRY` is optional. `IMAGE` is required. `TAG` is optional. `DIGEST` is optional, but strongly recommended.
+
+- If `IMAGE` is not specified, the runtime MUST fail with an error.
+- When `DIGEST` is specified, the runtime MUST use the version specified by that digest and MUST fail if an image of that digest cannot be found.
+- When no `DIGEST` is specified, but a `TAG` is specified, the runtime MUST use an image with that tag or fail.
+- When neither `DIGEST` nor `TAG` is defined, the behavior is undefined.
```

Comment:
Behaviour of some other clients is to choose `:latest` - do we want to add a note on avoiding the use of the `latest` tag?

Reply:
Is there also an assumption that docker.io is the default registry or do we want to show a fully qualified image with tag and the recommended way from a spec perspective?

Reply:
To be consistent with existing registries and registry clients behavior, we should default to the `:latest` tag.

Reply:
@sajayantony can you open a separate issue for that?

Reply:
Sure my comment is non-blocking. And will open another item to track this.